motor_control/throttle/base_throttle.h

```diff
@@ -1,21 +1,36 @@
 #ifndef BASE_THR_H
 #define BASE_THR_H
 
+//#include <Servo.h>
+#define throttleIn 5   // Placeholders
+#define ESCOutput 3
+#define button 7
+
 class Throttle
 {
 public:
-    Throttle(const int &pin);
+    Throttle();
 
-    virtual void compute_motor_value() = 0; // this needs to be implemented differently for variable/button throttle
+    void run();
+    void setup();
+    void loop();
```

Comment:
The point of this class and the inherited class is to be a library of functions, and not necessarily a binary that is executable. As such, we do not need these arduino functions here. The main routine will be executing these, likely from the motor control code.